chall03/ksappi.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+  "fmt"
+  "net/http"
+  "io/ioutil"
+  "strings"
+  "strconv"
+)
+
+func main() {
+  resp, err := http.Get("https://chall03.hive.fi/")
+  if err != nil {
+    return
+  }
+  body, err := ioutil.ReadAll(resp.Body)
+  if err != nil {
+    return
+  }
+  splitted := strings.Split(string(body), ",")
+
+  red, err := strconv.Atoi(strings.Split(splitted[1], "=")[1])
+  green, err := strconv.Atoi(strings.Split(splitted[2], "=")[1])
+  blue, err := strconv.Atoi(strings.Split(strings.Split(splitted[3], "=")[1], " ")[0])
```

Comment:
err should be checked after usage.

and in Go if you are not using a return value you can use the **blank identifier** `_`

e.g 

```go
red, _ := strconv.Atoi()
```